tests/test_otomi_morphology.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from elotl.otomi.morphology import Analyser
+from json import JSONEncoder
+import json
+
+
+class TokenEncoder(JSONEncoder):
+	def default(self, o):
+		return o.__dict__
+
+class TestAnalysis(unittest.TestCase):
+	analyser = Analyser()
+
+	def test_analysis_tokenisation(self):
```

Comment:
I suggest to make fail cases in tests, but I think it's ok by now

Reply:
This is my first time writing unit tests so I'm not sure what that means, but @Lguyogiro any thoughts?

Reply:
@umoqnier by "fail cases" do you mean (1)  tests that test when the code should throw an exception, or (2) unittests that define future desired functionality, that we expect to fail currently?

Either way, I agree that for now this PR LGTM.

One work in progress item we have is the Otomí interlinear glossing system. We should begin thinking about how to support both that and this analyzer (since presumably the glosser will also include an analysis).